src/app/helptext/task-calendar/replication/replication-wizard.ts

```diff
@@ -4,19 +4,30 @@ export default {
     step1_label: T('What and Where'),
 
     exist_replication_placeholder: T('Load Previous Replication Task'),
-    exist_replication_tooltip: T(''),
+    exist_replication_tooltip: T('Fill the wizard with the settings from \
+ a saved replication.'),
 
     source_datasets_from_placeholder: T('Source Datasets'),
-    source_datasets_from_tooltip: T(''),
+    source_datasets_from_tooltip: T('Datasets that have snapshots to \
+ replicate. Define the location of the source datasets, then select \
```

Comment:
"Define the location" is unclear.  "Define" is almost always not what users do--they choose or select or set.
```suggestion
 replicate. Select  \
```